src/Apps/gAtmoEvGen.cxx

```diff
@@ -359,6 +370,10 @@ int main(int argc, char** argv)
   mcj_driver->UseGeomAnalyzer(geom_driver);
   mcj_driver->Configure();
   mcj_driver->UseSplines();
+  /* Note: For the method of calculating the total number of events using a
+   * livetime we *need* to force a single probability scale. So if you change
+   * the next line you need to make sure that the user didn't specify the -T
+   * option. */
   mcj_driver->ForceSingleProbScale();
```

Comment:
I presume it's thus safe to call `ForceSingleProbScale()` in all cases, but _unsafe_ to *not* call it if `-T` was specified.  Am I reading that right?

Reply:
Yes that's right. Do you have any suggestions on how I can update that comment or if it belongs somewhere else? I added that just as kind of a future warning to anyone editing it that they shouldn't change that without adding a guard to check against the -T option.